new-website/src/3.x/docs/_meta.json

```diff
@@ -38,6 +38,11 @@
     "name": "migration-guides",
     "label": "Migration Guides"
   },
+  {
+    "type": "file",
+    "name": "repack-init",
+    "label": "repack-init cli"
+  },
```

Comment:
lowercase entry in the sidebar menu looks weird, lets hide it inside of a directory named: `CLI commands` and there have an entry of `repack-init`.